mars/config.py

```diff
@@ -379,6 +379,7 @@ def validate(x):
 default_options.register_option('optimize.default_disk_io_speed', 10 * 1024 ** 2, validator=is_integer)
 
 default_options.register_option('optimize_tileable_graph', True, validator=is_bool)
+default_options.register_option('optimize.head_limit', 1000, validator=is_integer)
```

Comment:
Rename to ``head_optimize_limit`` and move to the line below ``'optimize.default_disk_io_speed'``.